src/types/mod.rs

```diff
@@ -23,19 +23,29 @@ pub struct Eip712Fee {
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 #[serde(rename_all = "camelCase")]
 pub struct BridgeAddresses {
+    /// The address of the default shared bridge on Layer 1.
     pub l1_shared_default_bridge: Option<Address>,
+    /// The address of the default shared bridge on Layer 2.
     pub l2_shared_default_bridge: Option<Address>,
+    /// The address of the default ERC-20 bridge on Layer 1.
     pub l1_erc20_default_bridge: Option<Address>,
+    /// The address of the default ERC-20 bridge on Layer 2.
     pub l2_erc20_default_bridge: Option<Address>,
+    /// The address of the Wrapped Ethereum (WETH) bridge on Layer 1.
     pub l1_weth_bridge: Option<Address>,
+    /// The address of the Wrapped Ethereum (WETH) bridge on Layer 2.
     pub l2_weth_bridge: Option<Address>,
+    /// The address of the legacy shared bridge on Layer 2.
     pub l2_legacy_shared_bridge: Option<Address>,
```

Comment:
Probably it would be cool to mention under which circumstances these can be `None` (AFAIU some fields are deprecated, and some are only set in newer versions), but it would require some research. You can ping Stas for that.

However, it's OK if we postpone it.